### ISSUE TITLE
Language name

### ISSUE THREAD

**Issue by aunkrig** [ONorg/Javon#1]
Names matter... take your time to pick the right one!

However I discourage names like "Javon", because ORACLE hold a trademark on the name "Java", and obviously they do not allow people to name languages similarly (SCALA, XTEND, ...).

A name with, instead, a single "J" in it would definitely make sense to indicate the "proximity" to the Java ecosystem. To my mind come "Emerj", "June", "Jessir" ("Yes-Sir"), "Jeder" (german for "everybody"), "Julep".

**Comment by lyrachord** [comment]
Yes, Thanks. You are right. Names do matter. There is a saying, "Name Illegal, Nor Speach Logical".

Indeed, I dislike "Javon" as well. It is here just because it's a Java binding of ON language. About ON, it's a long story. :)

Inspired your kind advices, I think the two names "LONG" or "GONG" may be not bad.

Thank you again.
